### PR TITLE
14346 map card zero case

### PR DESCRIPTION
**Pull request by albertoac**
Added Initial state for the maps page

### CODE REVIEW

File: lib/assets/javascripts/new-dashboard/pages/Maps.vue [hunk 1]
```diff
@@ -15,22 +15,34 @@
...
         </template>
       </SectionTitle>
-    </div>
 
-    <ul class="grid" v-if="isFetchingMaps">
-      <li class="grid-cell grid-cell--col4 grid-cell--col6--tablet grid-cell--col12--mobile" v-for="n in 12" :key="n">
-        <MapCardFake></MapCardFake>
-      </li>
-    </ul>
+      <InitialState title="Create your first map to predict key insights" v-if="!isFetchingMaps && numResults <= 0">
```
Comment (jesusbotella) [inline lib/assets/javascripts/new-dashboard/pages/Maps.vue]: Let's put all texts under i18n translations :)

File: lib/assets/javascripts/new-dashboard/pages/Maps.vue [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <template>
 <section class="section">
   <div class="maps-list-container container grid">
-    <div class="grid-cell grid-cell--col12">
-      <SectionTitle title='Your Maps' description="This is a description test">
+    <div class="grid-cell--col12">
+      <SectionTitle class="grid-cell" title='Your Maps' description="This is a description test">
```
Comment (jesusbotella) [inline lib/assets/javascripts/new-dashboard/pages/Maps.vue]: Why this change?

Comment (albertoac) [reply lib/assets/javascripts/new-dashboard/pages/Maps.vue]: If we apply the grid-cell to the parent, then the cards will have an extra 20px space in the sides. Removing the grid-cell from the parent and putting it directly on the element, fixes that issue

**Comment by jesusbotella** [comment]
retest this please

**Comment by jesusbotella** [comment]
retest this please
